tensorflow/lite/micro/cortex_m_corstone_300/README.md

```diff
@@ -0,0 +1,35 @@
+
+# Running a fixed virtual platform based on Corstone-300 software
+
+This target makes use of a fixed virtual platform (FVP) based on Arm Cortex-300 based software.
+More info about Arm Corstone-300 software: https://developer.arm.com/ip-products/subsystem/corstone/corstone-300
+More info about FVPs: https://developer.arm.com/tools-and-software/simulation-models/fixed-virtual-platforms
```

Comment:
these render without newlines in markdown. If you can make sure the markdown is rendered cleanly, that would be great.

Reply:
Yes will do